config/data/langs.toml

```diff
@@ -22,6 +22,24 @@ each argument is NULL terminated.
 x is a no-op.
 '''
 
+[APL]
+args       = [ '/usr/bin/apl', '-' ]
+experiment = 135
+size       = '14.8 MiB'
+version    = 'Dyalog APL 18.2.45405'
```

Comment:
I couldn't get 19.0.48958 to be as well-functional as the current implementation. But then again, I invite you to give `latest-langs` a run.

Reply:
I did now, and it fixed all my shit.